legacy/setup-debian.sh

```diff
@@ -4,11 +4,12 @@ export DEBIAN_FRONTEND=noninteractive
 apt-get update
 apt-get -y upgrade
 apt-get -y install autoconf automake binutils \
-bison build-essential bzip2 ca-certificates cmake cmake-data coreutils \
+bison build-essential bzip2 ca-certificates cmake-data coreutils \
 curl debianutils file findutils flex g++ gcc gfortran git gzip \
 hostname libbz2-dev libcurl4-openssl-dev libicu-dev libgl1-mesa-dev libglu1-mesa-dev \
 libncurses-dev libssl-dev libtool libx11-dev libxerces-c-dev libxext-dev libxft-dev \
 libxml2-dev libxmu-dev libxpm-dev libyaml-cpp-dev lsb-release make patch python-dev \
-python3-dev sed subversion tar unzip wget xutils-dev xz-utils
+python3-dev rsync sed subversion tar unzip wget xutils-dev xz-utils
+apt-get -t buster-backports install cmake
```

Comment:
Doesn't this also require some config to enable the backports archives?

In any case, if we hard-code `buster`, let's duplicate the file as `setup-debian-10.sh`.

Reply:
This commands for backports works out of the box (to my own surprise :) ).
And yes, it's wise to mention it's for debian 10 only
Note that the rsync requirement is generic, though.

Reply:
Nope, backports need to be enabled for this to work:
```console
root@lxci002:~# lsb_release -d
Description:    Debian GNU/Linux 10 (buster)
root@lxci002:~# apt-get -t buster-backports install cmake
Reading package lists... Done
E: The value 'buster-backports' is invalid for APT::Default-Release as such a release is not available in the sources
```

So if you do this (which is okay with me), please consider:
* Renaming setup-debian to setup-debian-10. We support Debian in version 10 only, currently. (and check the references to it, and update them as needed).
* Checking the docs, and add a note about backports should be enabled. Possibly linking to backports.debian.org.

Reply:
I integrated your suggestions but I'm completly lost within the interface. I have no idea what to do next. Sorry for being a noob.